rhoas/provider.go

```diff
@@ -93,11 +94,14 @@ func providerConfigure(ctx context.Context, d *schema.ResourceData) (interface{}
 		os.Exit(1)
 	}
 
-	// nolint: contextcheck
-	httpClient := authAPI.BuildAuthenticatedHTTPClient(d.Get("offline_token").(string))
-
 	localDevelopmentServer := os.Getenv(LocalDevelopmentEnv)
 
+	httpClient := &http.Client{}
+	if localDevelopmentServer == "" {
```

Comment:
LGTM it is exactly what I'm doing with Ansible.
I'm unsure if `""` will suffice? For Ansible the variable is `None` type if not passed through. I'm not sure how TF works so you may be right. just something to keep an eye out for

Reply:
This is Go, here there is no `None` :) Also, as `localDevelopmentServer` is a string, `""` would be the `nil` value.